common/bitcointree/type.go

```diff
@@ -1,6 +1,6 @@
 package bitcointree
 
 type Receiver struct {
-	Pubkey string
+	Script VtxoScript
```

Comment:
why not using a more high-level `Descriptor string` and then we translate it to vtxo script by calling the parse func?